stable/filebeat/README.md

```diff
@@ -40,8 +41,10 @@ The following table lists the configurable parameters of the filebeat chart and
 | `config.http.enabled`                                    |                                                                                                          | `false`                                            |
 | `config.http.port`                                       |                                                                                                          | `5066`                                             |
 | `indexTemplateLoad`                                      | List of Elasticsearch hosts to load index template, when logstash output is used                         | `[]`                                               |
+| `command`                                                | Custom command (Docker Entrypoint)                                                                       | `[]`                                               |
+| `args`                                                   | Custom args (Docker Cmd)                                                                                 | `[]`                                               |
 | `plugins`                                                | List of beat plugins                                                                                     | `[]`                                               |
-| `extraVars`                                              | A map of additional environment variables                                                                | `{}`                                               |
+| `extraVars`                                              | A list of additional environment variables                                                                | `[]`                                              |
```

Comment:
this is a breaking change as the format changes - is it worth it?
if it is - it means a new major version of the chart